static/src/js/components/PermissionList/PermissionList.jsx

```diff
@@ -107,6 +141,37 @@ const PermissionList = () => {
     }
   ]
 
+  const [deleteAclMutation] = useMutation(DELETE_ACL)
```

Comment:
Need to double check with my current ticket, do we need a break any caches after this mutation?